README.md

```diff
@@ -28,10 +41,17 @@ af.init(appId, devKey, function(err, events) {
     af.sendTrackingWithEvent(eventName, {key1: value1, key2: value2}, function(err, events) {
         //events is {key1: value1, key2, value2}
     });
+})
+```
 
-    //To get appsFlyerUID an event,
-    af.getAppsFlyerUID(function(err, uid) {
```

Comment:
Oh dang. This was my bad. I removed a bit too much of the usage. getAppsFlyerUID is still working in ios but not yet supported in my android with my changes.